common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -5585,7 +5585,7 @@ public static enum ConfVars {
     HIVE_QUERY_REEXECUTION_ENABLED("hive.query.reexecution.enabled", true,
         "Enable query reexecutions"),
     HIVE_QUERY_REEXECUTION_STRATEGIES("hive.query.reexecution.strategies",
-        "overlay,reoptimize,reexecute_lost_am,dagsubmit,recompile_without_cbo",
+        "overlay,reoptimize,reexecute_lost_am,dagsubmit,recompile_without_cbo,optimistic_concurrency",
```

Comment:
that is not optimistic concurrency, should we name it `retry_on_conflict`?

Reply:
done